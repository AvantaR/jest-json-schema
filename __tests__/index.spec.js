const matchers = require('../index.js');

describe('index', () => {
  it('should export all the matchers from the matchers directory', () => {
    expect(matchers).toMatchSnapshot();
  });
});
