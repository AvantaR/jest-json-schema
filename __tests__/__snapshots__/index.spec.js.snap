// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index matchersWithFormats should return all the matchers 1`] = `
Object {
  "toMatchSchema": [Function],
}
`;

exports[`index should export all the matchers from the matchers directory 1`] = `
Object {
  "toMatchSchema": [Function],
}
`;
